src/App.js

```diff
@@ -6,7 +6,7 @@ function App() {
 	return (
 		<AppLoader
 			token={process.env.REACT_APP_TOKEN}
-			instance={{ZUID: `${process.env.REACT_APP_ZUID}`}}>
+			instance={{ZUID: `${process.env.REACT_APP_ZUID}`, ID : 8355630}}>
```

Comment:
@allenpigar the `8355630` number is hard written

Reply:
added intance id on .env

![image](https://user-images.githubusercontent.com/50983144/178478147-df39a524-bf61-4a12-bb9b-d6a9242b080e.png)


![image](https://user-images.githubusercontent.com/50983144/178478206-c98e828e-337b-4fd9-9744-b0ddc3dc046f.png)